### PR TITLE
Update packages and close session correctly when still on event loop 

### PR DESCRIPTION
**Pull request by lahariganti**
Swift package is failing to work well (`swift package update`) as version requirements of cross dependencies are not compatible.  

https://github.com/Kitura/SwiftyRequest/compare/master...mbarnach:SwiftyRequest:master

### CODE REVIEW

File: .swiftpm/xcode/package.xcworkspace/xcuserdata/lahariganti.xcuserdatad/UserInterfaceState.xcuserstate
Comment (mbarnach) [inline .swiftpm/xcode/package.xcworkspace/xcuserdata/lahariganti.xcuserdatad/UserInterfaceState.xcuserstate:1]: Is this file needed?

File: .swiftpm/xcode/xcuserdata/lahariganti.xcuserdatad/xcschemes/xcschememanagement.plist
Comment (mbarnach) [inline .swiftpm/xcode/xcuserdata/lahariganti.xcuserdatad/xcschemes/xcschememanagement.plist:1]: Is this file needed?

File: .swiftpm/xcode/package.xcworkspace/xcuserdata/lahariganti.xcuserdatad/UserInterfaceState.xcuserstate
Comment (lahariganti) [reply .swiftpm/xcode/package.xcworkspace/xcuserdata/lahariganti.xcuserdatad/UserInterfaceState.xcuserstate:1]: No

File: .swiftpm/xcode/xcuserdata/lahariganti.xcuserdatad/xcschemes/xcschememanagement.plist
Comment (lahariganti) [reply .swiftpm/xcode/xcuserdata/lahariganti.xcuserdatad/xcschemes/xcschememanagement.plist:1]: No, removed.

File: Package.swift [hunk 1]
```diff
@@ -27,10 +27,10 @@ let package = Package(
         )
     ],
     dependencies: [
```
Comment (lahariganti) [inline Package.swift:29]: Change to remote @lahariganti

Comment (lahariganti) [reply Package.swift:29]: https://github.com/Kitura/CircuitBreaker/pull/58/files

**Review by dannys42 — APPROVED** [review]
